src/index.html

```diff
@@ -5,9 +5,127 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.css">
+    <link rel="icon" href="images/favicon.png">
+    <link href="https://fonts.googleapis.com/css?family=Ubuntu:300,400,400i,500,700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
-    <script type="text/javascript" src="scripts/main.js"></script>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone">+1 234 555-55-55</div>
+        <div class="header__action"><a href="#" class="header__link">Call to order </a></div>
+      </div>
+      <nav class="nav header__nav">
+        <ul class="nav__list">
+          <li class="nav__item nav__item--active">Home</li>
+          <li class="nav__item"><a href="#about" class="nav__link">About us</a></li>
+          <li class="nav__item"><a href="#gallery" class="nav__link">Gallery</a></li>
+          <li class="nav__item"><a href="#consultation" class="nav__link">Consultation</a></li>
+          <li class="nav__item"><a href="#testimonials" class="nav__link">Testimonials</a></li>
+          <li class="nav__item"><a href="#contacts" class="nav__link">Contacts</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="section">
+        <div class="info" id="about">
+          <h1 class="info__heading">Miami <strong>best</strong> Real estate <a href="#" class="link">here</a></h1>
+          <div class="info__text">
+            <p><strong>50 years experience in love.</strong><br>Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.</p>
```

Comment:
переносы для читаемости
```suggestion
            <p>
              <strong>50 years experience in love.</strong><br>
              Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
            </p>
```

Reply:
Сделал переносы здесь и дальше по коду